sei-incentivized-testnet/gentx/sample-gentx.json

```diff
@@ -1 +0,0 @@
ges":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","commission":{"max_change_rate":"0.010000000000000000","max_rate":"0.200000000000000000","rate":"0.100000000000000000"},"delegator_address":"sei1wvwwcq0723nay8h06hlnjftr4y63k25rqezjhk","description":{"details":"","identity":"","moniker":"cyson-validator","security_contact":"","website":""},"min_self_delegation":"1","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"Hi+rgvDXTeuxR9IHaqwGL2cerPQp3kXMcVU9WbpYfHc="},"validator_address":"seivaloper1wvwwcq0723nay8h06hlnjftr4y63k25r7knukx","value":{"amount":"70000000000000000000","denom":"usei"}}],"non_critical_extension_options":[],"timeout_height":"0"},"signatures":["8UWuEE9hUj7f6UGU+PDUyB2u6NRm+rVjPj2kzQX2BgBd7/DtA316+JPAAPWEJBr1ao5RJ8aM3cjSrMd+rbfONg=="]}]},"gov":{"deposit_params":{"max_deposit_period":"172800s","min_deposit":[{"amount":"10000000","denom":"usei"}]},"deposits":[],"proposals":[],"starting_proposal_id":"1","tally_params":{"quorum":"0.334000000000000000","threshold":"0.500000000000000000","veto_threshold":"0.334000000000000000"},"votes":[],"voting_params":{"voting_period":"30s"}},"ibc":{"channel_genesis":{"ack_sequences":[],"acknowledgements":[],"channels":[],"commitments":[],"next_channel_sequence":"0","receipts":[],"recv_sequences":[],"send_sequences":[]},"client_genesis":{"clients":[],"clients_consensus":[],"clients_metadata":[],"create_localhost":false,"next_client_sequence":"0","params":{"allowed_clients":["06-solomachine","07-tendermint"]}},"connection_genesis":{"client_connection_paths":[],"connections":[],"next_connection_sequence":"0","params":{"max_expected_time_per_block":"30000000000"}}},"mint":{"minter":{"annual_provisions":"0.000000000000000000","inflation":"0.130000000000000000"},"params":{"blocks_per_year":"6311520","goal_bonded":"0.670000000000000000","inflation_max":"0.200000000000000000","inflation_min":"0.070000000000000000","inflation_rate_change":"0.130000000000000000","mint_denom":"usei"}},"params":null,"slashing":{"missed_blocks":[],"params":{"downtime_jail_duration":"600s","min_signed_per_window":"0.500000000000000000","signed_blocks_window":"100","slash_fraction_double_sign":"0.050000000000000000","slash_fraction_downtime":"0.010000000000000000"},"signing_infos":[]},"staking":{"delegations":[],"exported":false,"last_total_power":"0","last_validator_powers":[],"params":{"bond_denom":"usei","historical_entries":10000,"max_entries":7,"max_validators":100,"unbonding_time":"1814400s"},"redelegations":[],"unbonding_delegations":[],"validators":[]},"transfer":{"denom_traces":[],"params":{"receive_enabled":true,"send_enabled":true},"port_id":"transfer"},"upgrade":{},"vesting":{},"wasm":{"codes":[],"contracts":[],"gen_msgs":[],"params":{"code_upload_access":{"address":"","permission":"Everybody"},"instantiate_default_permission":"Everybody"},"sequences":[]}},"chain_id":"sei-chain","gentxs_dir":"/Users/cyson/.sei/config/gentx","moniker":"cyson-validator","node_id":"2e1b15d6a92596a5eb656f2e04345b5d158a58cd"}
```

Comment:
@LCyson removing this since i don't think we need it and it will confuse folks / introduce errors when we bring the chain up